docs/docs/reference/crd-reference/config.md

```diff
@@ -49,7 +51,10 @@ Each cluster should have a single `KeptnConfig` CRD that describes all configura
 ### OTel example
 
 This example specifies the URL of the OpenTelemetry collector
-and that the automatic app discovery should be run every 40 seconds:
+and that the automatic app discovery should be run every 40 seconds.
+Additionally the CloudEvents endpoint URL is specified and the
+blocking of the deployment of the application is disabled in case
+of the pre-deployment task or evaluation failure.
```

Comment:
maybe we can make use of annotations to explain line by line what is happening here

Reply:
https://squidfunk.github.io/mkdocs-material/reference/annotations/

Reply:
what do you mean here ?

Reply:
adapted